keyboards/era/linx3/fave65s/keyboard.json

```diff
@@ -0,0 +1,197 @@
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "rainbow_moving_chevron": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "dual_beacon": true,
+            "rainbow_beacon": true,
+            "rainbow_pinwheels": true,
+            "flower_blooming": true,
+            "raindrops": true,
+            "jellybean_raindrops": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "pixel_fractal": true,
+            "pixel_flow": true,
+            "pixel_rain": true,
+            "starlight": true,
+            "starlight_dual_hue": true,
+            "starlight_dual_sat": true,
+            "riverflow": true
+        },
+        "default": {
+            "animation": "rainbow_moving_chevron"
+        },
+        "driver": "ws2812",
+        "layout": [
+            {"matrix": [2, 0], "x": 0, "y": 0, "flags": 8},
+            {"x": 0, "y": 0, "flags": 2},
+            {"x": 13, "y": 64, "flags": 2},
+            {"x": 26, "y": 64, "flags": 2},
+            {"x": 40, "y": 64, "flags": 2},
+            {"x": 53, "y": 64, "flags": 2},
+            {"x": 66, "y": 64, "flags": 2},
+            {"x": 79, "y": 64, "flags": 2},
+            {"x": 92, "y": 64, "flags": 2},
+            {"x": 105, "y": 64, "flags": 2},
+            {"x": 119, "y": 64, "flags": 2},
+            {"x": 132, "y": 64, "flags": 2},
+            {"x": 145, "y": 64, "flags": 2},
+            {"x": 158, "y": 64, "flags": 2},
+            {"x": 171, "y": 64, "flags": 2},
+            {"x": 184, "y": 64, "flags": 2},
+            {"x": 198, "y": 64, "flags": 2},
+            {"x": 211, "y": 64, "flags": 2},
+            {"x": 224, "y": 64, "flags": 2},
+            {"x": 224, "y": 13, "flags": 2},
+            {"x": 224, "y": 26, "flags": 2},
+            {"x": 224, "y": 38, "flags": 2},
+            {"x": 224, "y": 51, "flags": 2},
+            {"x": 224, "y": 64, "flags": 2},
+            {"x": 211, "y": 64, "flags": 2},
+            {"x": 198, "y": 64, "flags": 2},
+            {"x": 184, "y": 64, "flags": 2},
+            {"x": 171, "y": 64, "flags": 2},
+            {"x": 158, "y": 64, "flags": 2},
+            {"x": 145, "y": 64, "flags": 2},
+            {"x": 132, "y": 64, "flags": 2},
+            {"x": 119, "y": 64, "flags": 2},
+            {"x": 105, "y": 64, "flags": 2},
+            {"x": 92, "y": 64, "flags": 2},
+            {"x": 79, "y": 64, "flags": 2},
+            {"x": 66, "y": 64, "flags": 2},
+            {"x": 53, "y": 64, "flags": 2},
+            {"x": 40, "y": 64, "flags": 2},
+            {"x": 26, "y": 64, "flags": 2},
+            {"x": 13, "y": 64, "flags": 2},
+            {"x": 0, "y": 64, "flags": 2},
+            {"x": 0, "y": 51, "flags": 2},
+            {"x": 0, "y": 38, "flags": 2},
+            {"x": 0, "y": 26, "flags": 2},
+            {"x": 0, "y": 13, "flags": 2}
```

Comment:
This LED layout looks incorrect. There are multiple entries with the same position, which is not physically possible.
![image](https://github.com/qmk/qmk_firmware/assets/4781841/2d5a584e-4704-43f0-a7af-861073c750fa)

Reply:
Oh thank you so much!
I hadn't noticed this.

It has now been corrected.

Reply:
![image](https://github.com/qmk/qmk_firmware/assets/4781841/0954f056-da55-484e-819a-1f2e05fce3bd)
👍

Reply:
:) Thank you for your help!